vector/src/main/java/im/vector/app/features/onboarding/ftueauth/FtueAuthVariant.kt

```diff
@@ -399,15 +390,11 @@ class FtueAuthVariant(
     }
 
     private fun onAccountCreated() {
-        if (vectorFeatures.isOnboardingPersonalizeEnabled()) {
-            activity.supportFragmentManager.popBackStack(null, FragmentManager.POP_BACK_STACK_INCLUSIVE)
-            activity.replaceFragment(
-                    views.loginFragmentContainer,
-                    FtueAuthAccountCreatedFragment::class.java,
-            )
-        } else {
-            navigateToHome(createdAccount = true)
-        }
+        activity.supportFragmentManager.popBackStack(null, FragmentManager.POP_BACK_STACK_INCLUSIVE)
```

Comment:
Just an idea, it would be cool if we could encapsulate all navigation code into a dedicated interface. It abstracts the navigation implementation and reduce the component responsability. Of course I don't suggest to change it in this PR 😄 But maybe later?

Reply:
we have one for activities but not fragments, I think it's great idea and would help towards making this class testable

I've tracked it as an issue https://github.com/vector-im/element-android/issues/5508